app/assets/javascripts/split_tabs.js

```diff
@@ -0,0 +1,29 @@
+$(function() {
+  var splitTabsContainer = ".split-tabs";
+  var splitTab = ".split-tab";
+  var splitTabPanel = ".split-tab-panel";
+  var activeClass = "is-active";
+  var transitionTime = 300;
+
+  $(splitTab + ":first-child")
+    .addClass(activeClass)
+    .closest(splitTabsContainer)
+    .siblings(splitTabPanel + ":first-of-type")
+    .show();
+
+  $(splitTab).click(function() {
+    $(this)
+      .siblings(splitTab)
+      .removeClass(activeClass)
+      .closest(splitTabsContainer)
+      .siblings(splitTabPanel)
+      .hide();
+
+    $(this)
+      .addClass(activeClass)
+      .closest(splitTabsContainer)
+      .siblings(splitTabPanel)
+      .eq($(this).index(splitTab))
+      .fadeIn(transitionTime);
+  });
```

Comment:
I think we can simplify this a bit. I always try to avoid using `sibling` and `closest` because it relies on the markup not changing. One way to avoid this is to use `data-attributes`. For example, the matched tab could have `data-associated-panel: "matched"` and then the associated panel might have `data-panel: "matched"`. Then you might do the following jQuery:

```javascript
$(splitTab).click(function() {
  var associatedPanel = $(this).data("associated-panel");

  splitTabPanel.hide();
  $("[data-panel="+associatedPanel+"]").show();
});
```

Obviously there's a bit more to your code than just what I wrote above, but just wanted to give you a taste.